ISISComputingGroup/ibex_gui
# Ticket1374 toggle experiment title visibility

NOTE: While working on this ticket I have discovered that the PVManager does not properly handle writing values of type Boolean on ChannelAccess, which should be written as a numeric values 0/1 (e.g. using LongChannel) instead. We should find an appropriate place to record this for future reference.

###Description of work

Added a checkbox to the DAE Run Summary Tab that is wired up to a newly created PV in the ISISDAE IOC intended to be used for toggling visibility of the Run Title in the dataweb dashboard page.

###To test

Fixes https://github.com/ISISComputingGroup/IBEX/issues/1374
To Test: 
1. run ISISDAE IOC (make sure you have the latest version including the newly added PV, see comments)
2. camonitor %instrument prefix%:DAE:TITLE:DISPLAY
3. check / uncheck the box in the DAE Run Summary Tab of the IBEX GUI.

###Acceptance criteria

The Checkbox accurately reflects the state of the TITLE:DISPLAY PV and vice versa.

---

#### Code Review


### Functional Tests
